### PR TITLE
Fixed FindBugs issue in PhoneHome

### PR DESCRIPTION
**Pull request by Donnerbart**
* added `EmptyStatement.ignore()` to make FindBugs happy again
* little cleanup of `PhoneHome`
* increased code coverage of `PhoneHome`

Code coverage is up from 75% methods, 60% lines to 87% methods, 76% lines covered.

### CODE REVIEW

File: hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java [hunk 1]
```diff
@@ -253,34 +258,36 @@ private void addManCenterInfo(Node hazelcastNode, int clusterSize, PhoneHomePara
...
+
             // if management center is not running,
             // connection.getInputStream() throws 'java.net.ConnectException: Connection refused'
             inputStream = connection.getInputStream();
-            responseCode = connection.getResponseCode();
             reader = new InputStreamReader(inputStream, "UTF-8");
-            JsonObject mcPhoneHomeInfoJson = JsonValue.readFrom(reader).asObject();
-            version = JsonUtil.getString(mcPhoneHomeInfoJson, "mcVersion");
-            license = JsonUtil.getString(mcPhoneHomeInfoJson, "mcLicense", null);
+            JsonObject mcPhoneHomeInfoJson = Json.parse(reader).asObject();
```
Comment (Donnerbart) [inline hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java]: `JsonValue.readFrom(Reader)` was deprecated, `Json.parse(Reader)` was proposed as replacement in its JavaDoc.

File: hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java [hunk 2]
```diff
@@ -253,34 +258,36 @@ private void addManCenterInfo(Node hazelcastNode, int clusterSize, PhoneHomePara
...
+            String version = getString(mcPhoneHomeInfoJson, "mcVersion");
+            String license = getString(mcPhoneHomeInfoJson, "mcLicense", null);
+
+            int responseCode = connection.getResponseCode();
+            if (responseCode == HttpURLConnection.HTTP_OK) {
```
Comment (hasancelik) [inline hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java]: There is a behavioural changes so these changes affect meaning of the `mcver` and `mclicense` infos for the marketing team. You can find detailed informations about statuses at related PRD:
https://hazelcast.atlassian.net/wiki/spaces/PM/pages/227148329/Improve+Phone+Home+Data+for+ManCenter+Licenses+TDD

Comment (Donnerbart) [reply hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java]: I've reverted the order. That part was indeed untested, so we cannot very the exact behavior on different error cases.

Good catch!

**Comment by Holmistr** [comment]
Hey @hasancelik, @Donnerbart updated the PR according to your comment, can you review again?

**Pull request merged**